tests/data_context/test_data_context.py

```diff
@@ -1032,24 +1035,24 @@ def test_data_context_create_does_not_raise_error_or_warning_if_ge_dir_exists(
     tmp_path_factory,
 ):
     project_path = str(tmp_path_factory.mktemp("data_context"))
-    DataContext.create(project_path)
+    FileDataContext.create(project_path)
 
 
 @pytest.fixture()
 def empty_context(tmp_path_factory):
     project_path = str(tmp_path_factory.mktemp("data_context"))
-    DataContext.create(project_path)
+    FileDataContext.create(project_path)
     ge_dir = os.path.join(project_path, "great_expectations")
     assert os.path.isdir(ge_dir)
-    assert os.path.isfile(os.path.join(ge_dir, DataContext.GX_YML))
+    assert os.path.isfile(os.path.join(ge_dir, FileDataContext.GX_YML))
     context = DataContext(ge_dir)
-    assert isinstance(context, DataContext)
+    assert isinstance(context, FileDataContext)
```

Comment:
Can you annotate the type of this fixture?
Is it `pathlib.Path` or a `str`?

Reply:
it's a `pytest` built-in: `_pytest.fixture.TemporaryPathFactory` I believe